dfget/core/uploader/peer_server.go

```diff
@@ -457,7 +458,13 @@ func (ps *peerServer) shutdown() {
 	ps.syncTaskMap.Range(func(key, value interface{}) bool {
 		task, ok := value.(*taskConfig)
 		if ok {
-			ps.api.ServiceDown(task.superNode, task.taskID, task.cid)
+			resp, e := ps.api.ServiceDown(task.superNode, task.taskID, task.cid)
+			if e != nil {
+				logrus.Errorf("send ServiceDown error:%v", e)
```

Comment:
Same issue here, please see the above comments. Thanks.

Reply:
Done